src/components/Permissions/PermissionsTab.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import Page from 'components/Page'
+import { Tabs, Tab } from 'cozy-ui/transpiled/react/MuiTabs'
+import AppList from './AppList'
+import DataList from './DataList'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider/index'
+import { routes } from 'constants/routes'
+import { withRouter } from 'react-router-dom'
+import withAllLocales from '../../lib/withAllLocales'
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  }
+}
+const TabPanel = props => {
+  const { children, value, index, ...other } = props
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && children}
+    </div>
+  )
+}
+const PermissionsTab = ({ match, t }) => {
+  const slugOrData = match.params.page
+  const [tab, setTab] = useState(slugOrData)
+  const handleChange = (event, value) => setTab(value)
```

Comment:
@Noemie21 I'm still not happy with this implementation. We're copying a props to a state. We're creating duplication. And who says duplication, says synchronization problem. 

Can't we do something like that: 

```jsx
const PermissionsTab = () => {

const { t } = useI18n()
const { page } = useParams() // useParams is a hook coming from react-router 

const history = useHistory() // useHistory is a hook coming from react-router

const handleChange = (event, value) => history.push('/path/to/route/with/value)

return (
    <Page narrow>
      <Tabs value={page} onChange={handleChange}>
        <Tab
          value="slug"
          label={t('Permissions.applications')}
          href={`#${routes.appList}`}
          {...a11yProps(0)}
        />
        <Tab
          value="data"
          label={t('Permissions.data')}
          href={`#${routes.dataList}`}
          {...a11yProps(1)}
        />
      </Tabs>
      <Divider className="u-mb-1" />
      <TabPanel value={page} index="slug">
        <AppList />
      </TabPanel>
      <TabPanel value={page} index="data">
        <DataList />
      </TabPanel>
    </Page>
  )
}
```

Reply:
Like that, no more state involved, but you also get the back button of the browser working since you also change the URL.

Reply:
I replaced 

`const history = useHistory() // useHistory is a hook coming from react-router
const handleChange = (event, value) => history.push('/path/to/route/with/value)`

with 
``` const navigate = useNavigate()
  const handleChange = (event, value) => navigate(`/permissions/${value}`)```

is that ok ?

Reply:
Oh I didn't know about useNavigate()! Thanks

I see... This is a new API from react-router 6! Nice. 👏

Reply:
You get the idea about not using state since we don't need it. After that, the implementation is up to you. You are in a better position than me to write it ;)